src/control/server/drpc.go

```diff
@@ -43,21 +46,32 @@ func getDrpcClientConnection(sockDir string) *drpc.ClientConnection {
 	return drpc.NewClientConnection(clientSock)
 }
 
+func createSocketDir(sockDir string) error {
```

Comment:
Im ok with this for now but I have been debating if we should even be doing this. Since the socket directories are by default under a system controlled tmpfs maybe we just rely on the unit files or the rpm to make the right directories and we fail if its not there.

Reply:
yes I remember that discussion but was deferring making the decision as I don't think it's part of this patch